### ISSUE TITLE
Feature request - Option to switch between Confirmed and Existing cases in data

### ISSUE THREAD

**Issue by SergeS** [CSSEGISandData/COVID-19#128]
I would like to see how much existing cases are where, + would be nice to have it in charts

**Comment by CSSEGISandData** [comment]
We added a map for existing cases yesterday. It's a tab under the map. You can switch between cumulative case map and existing case map.
<img width="362" alt="Screen Shot 2020-02-25 at 12 11 20 PM" src="https://user-images.githubusercontent.com/60674295/75270080-f3e1ee80-57c7-11ea-9c94-9a033424216d.png">

**Comment by SergeS** [comment]
Yes I know, this is the reason I'm asking :-), I meant option to see same date in table on left side
